Comments

LoRayZm opened the pull request:
Use grunt build and grunt serve before
links : 
localhost/wiki/edit/name.of.the.plugin    -->  for the wiki editor
localhost/name.of.the.plugin/comments  --> for the comments
connected to #89
connected to #88

lorenzo-milesi committed:
[Wiki] Wiki Edition Page

DONE :
-> 1. Redirection to wiki edit page 	--- OK
-> 2. HTML + CSS view wiki edit page 	--- OK
NOT DONE :
-> 3. Add wiki edit to MongoDb

lorenzo-milesi committed:
[Wiki] Wiki Edition Page Update

Changing url

lorenzo-milesi committed:
[Wiki] Wiki and Wiki edit Page

DONE :
-> 1. Adding wiki param to class Plugin	--- Ok
-> 2. Adding wiki print to printplugin	---Ok
TO DO:
-> 3. Use theses classes in wiki edit / wiki show

lorenzo-milesi committed:
[Wiki] Adding Bootstrap Markdown Editor

DONE :
-> 1. Visual appearance	--- OK
-> 2. Updating Font Awesome --- OK
NOT DONE :
-> 3. Show a correct markdown preview

lorenzo-milesi committed:
[Wiki] On Bootstrap Markdown Editor

DONE PREVIOUSLY :
-> 1. Visual appearance	--- OK
-> 2. Updating Font Awesome --- OK
DONE :
-> 3. Show a correct markdown preview --- OK
TO DO :
-> 4. Update and/or record wiki in plugin

lorenzo-milesi committed:
[Wiki] New Url for Wiki Edit

lorenzo-milesi committed:
[Wiki] Try

lorenzo-milesi committed:
[Comments] Comments Page and Style

DONE :
1 -> Adding comments page redirection in views.py
2 -> comments.html page created
3 -> Giving a style to comments' text input
TO DO :
4 -> Create comments table in mongo and id to the plugin
5 -> Create Add / Delete / Display comments functions

lorenzo-milesi committed:
[Wiki] Edit wiki update

lorenzo-milesi committed:
[Wiki] Wiki EDIT IS WORKING !!

lorenzo-milesi committed:
[Comments] Changing routes

lorenzo-milesi committed:
[Comments] Catalog and Client views

lorenzo-milesi committed:
[Comments] Ajax Request

lorenzo-milesi committed:
[Comments] Big Upload Bug and all mongo data erased

lorenzo-milesi committed:
[Comments] User : comment system

lorenzo-milesi committed:
[Comments] CSS

lorenzo-milesi committed:
Merge branch 'wiki-edit' into comments

Conflicts:
	shuttleofx_catalog/Plugin.py
	shuttleofx_client/static_tmp/styles/main.less
	shuttleofx_client/views.py

lorenzo-milesi committed:
[Comments] synthax

Olivier-Daire commented on shuttleofx_client/static_tmp/styles/wiki.less:
```diff
@@ -0,0 +1,4 @@
+#metaWiki fieldset {
+    // border: 1px solid #2ab074;
```
Unused code ? Remove it :smile:

LoRayZm replied:
ah oui :p okey

lorenzo-milesi committed:
[Comments] Removal of useless files

lorenzo-milesi committed:
[Comments-Fix] A wild line had been erased !

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

Conflicts:
	shuttleofx_client/templates/layout.html

lorenzo-milesi committed:
[Wiki] Adding Wiki page and HREFS

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

Conflicts:
	shuttleofx_client/static_tmp/styles/main.less

lorenzo-milesi committed:
[Comments-Fix] It is aliive !!

KrisNumber24 commented:
It works !
But  : 
- in wiki, there is a Preview section at the bottom of the page but it displays nothing. Maybe you could delete this section.
- in comments, the display is strange, you have a list for the name and a list for the text...

Olivier-Daire commented:
Already said in the wiki branch that has been closed...
- when entering code inside ``` the indentation is not kept, and the syntax highlighting doesn't seems to work
- already said it but the preview button is not very well placed, in my mind, plus I think that there should be a change when you are in preview vs when you are in edit mode
- why is there a `<h3>Preview :</h3>` below the editor ? [EDIT] ok got it, it's when you save, but it think it kind of weird when you edit it
- a small padding on the left between the text and the input border would be great
- fullscreen mode is very cool ! But the preview button text goes black when we go into preview mode
- would be great if we could use `CTRL Z` but this may be complicated

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

lorenzo-milesi committed:
[Comments-Fix] CSS + User connected system

--> Modified way to record comments and names
--> Better CSS appearance
--> If the user is connected, his name is displayed automaticaly

ghost commented:
For the next update (coming soon)

Good news : 

---
- For the preview, I removed the button, instead we'll have a dynamic preview just below
- For the `<h3>Preview : </h3>`, it was a mistake, now it has been deleted
- A small padding on the left between the text and the border has been set
- Fullscreen mode has been updated
- `CTRL-Z` is already working :p

Bad news : 

---
- For ``` it is unfortunately normal, the developers themselves of Markdown Bootstrap and Markjown to Html do not use ``` for the same reasons,

lorenzo-milesi committed:
[Wiki Updates] New version

Updates :
_______________________
- Dynamic preview in Wiki Edit
- `<h3>Preview : </h3>`, deleted
- Small padding on the left between the text and the border set
- Fullscreen mode updated

KrisNumber24 commented:
Maybe you could change the behaviour of the button code in the editing wiki page to add four blank spaces to allow the users to indent their code.
I think you should change style of comments page because it appears really empty.

lorenzo-milesi committed:
[Comments] User image and Date

lorenzo-milesi committed:
[Wiki] Adding Button Indent

ghost commented:
Done !

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

lorenzo-milesi committed:
[Comments] Comments possible only if connected

lorenzo-milesi committed:
[Wiki] Only connected people can create or edit the wiki

lorenzo-milesi committed:
[Wiki] Last Wiki Edit

New :
Get and display user picture, user name and date at the bottom
of wiki

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

Olivier-Daire commented:
Nice ! :+1: 

Few things to improve : 
- **comment page** : change button wording to something more usual "send" or "comment" for example, change button's color
- **comment page** : try to display the date in a better way "16 Jan 2016" for example. At least remove the comma after the date

lorenzo-milesi committed:
[Wiki + Comments] CSS and Display

--> Changing CSS for messages (You must be logged in...)
    in comments and wiki
--> Changed button CSS and wording for submit comment
--> Display date of issue with Month Name
--> Fixed bug with new wiki

KrisNumber24 commented:
If it's possible, try to enhance code typing in the wiki page.

lorenzo-milesi committed:
[New Wiki Editor] Setting up for the wiki edit ver 2.0

--> Gruntfile.js modified : accepts folders in scripts/vendor/**/*.js
--> Add emoji from github
--> New .js and .css files needed for the new editor

lorenzo-milesi committed:
[New Wiki Editor] JS and CSS files

Adding .js and .css and .less files for wiki editor

lorenzo-milesi committed:
[New Wiki Editor] New Wiki Editor set up

lorenzo-milesi committed:
[New Wiki Editor] Downloading wiki in makedown or html

lorenzo-milesi committed:
[New Wiki Editor] New editor is working

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

lorenzo-milesi committed:
[New Wiki Editor] Edit some Css and HTML

lorenzo-milesi committed:
[New Wiki Editor - Fix] Fixed scrollbar over nav

lorenzo-milesi committed:
[Icons CSS]

lorenzo-milesi committed:
[Wiki Editor - Fix] Works eventually

lorenzo-milesi committed:
[New wiki Editor Fix] Items fixed

1. Wiki displayed for user logged in and not logged in
2. CSS and Layout for Wiki, Wiki Edit and Comments Pages
3. Download Markdown Button
4. Wrap-Word Fixed on wiki preview
5. CSS Glitch managed

lorenzo-milesi committed:
[Wiki Fix] Horizontal Scroll Glitch CSS

lorenzo-milesi committed:
Merge branch 'develop' of github.com:shuttleofx/ShuttleOFX into comments

Conflicts:
	shuttleofx_client/views.py

lorenzo-milesi committed:
[Wiki] Just one line modified
